modal/image.py

```diff
@@ -1418,7 +1420,7 @@ def conda(python_version: Optional[str] = None, force_build: bool = False):
             "`Image.conda` is deprecated."
             " Please use the faster and more reliable `Image.micromamba` constructor instead."
         )
-        deprecation_error((2024, 5, 2), message)
+        deprecation_error((2025, 5, 2), message)
```

Comment:
Hmm, this date is a few months in the future, unless you read it US-backwards? 😬